bigtable/client/client_options.h

```diff
@@ -53,6 +53,36 @@ class ClientOptions {
     return *this;
   }
 
+  void SetCompressionAlgorithm(grpc::ChannelArguments *channel_arguments,
+                               const grpc_compression_algorithm algorithm);
+
+  void SetGrpclbFallbackTimeout(grpc::ChannelArguments *channel_arguments,
+                                const int fallback_timeout);
+
+  void SetSocketMutator(grpc::ChannelArguments *channel_arguments,
+                        const grpc_socket_mutator* mutator);
+
+  void SetUserAgentPrefix(grpc::ChannelArguments *channel_arguments,
+                          const grpc::string& user_agent_prefix);
+
+  void SetResourceQuota(grpc::ChannelArguments *channel_arguments,
+                        const grpc::ResourceQuota& resource_quota);
```

Comment:
Some args have `&` attached to the type, some have it attached to the variable. Please be consistent.

Also, does clang-format enforce a particular way?

Reply:
I think the Google style is `DerivePointerAlignment: true`, which means the location is derived from other uses in the file.  I prefer `Foo* a` over `Foo *a` for what is worth.

Reply:
Yes, clang-format forces this one. Should I change it to Google style?

Reply:
Done.